tests/test_25_rt_nan_and_scanrate.py

```
@@ -0,0 +1,60 @@
+    scan_rate,
+    compute_qc_metrics,
+)
+
+
+def _spec(rt, level):
+    sp = oms.MSSpectrum()
+    sp.setRT(float(rt))
+    sp.setMSLevel(level)
+    sp.set_peaks((np.array([500.0]), np.array([100.0])))
+    if level >= 2:
+        prec = oms.Precursor(); prec.setMZ(500.0); prec.setCharge(2)
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)